test/Fixtures/CustomerFixtureTrait.php

```diff
@@ -122,12 +124,20 @@ public function getMinimalRegisteredB2bCustomer(): Customer
      */
     public function getMaximalRegisteredB2bCustomer(): Customer
     {
-        return $this->getMinimalRegisteredB2bCustomer()
+        $owner = (new CompanyOwner())
+            ->setFirstname('firstname')
+            ->setLastname('lastname')
+            ->setBirthdate('1999-01-01');
```

Comment:
DateTime as defined in Tech Spec.
Is there a reason as to why not using DateTime?